pkg/cri/sbserver/podsandbox/controller.go

```diff
@@ -84,8 +85,23 @@ func New(
 var _ sandbox.Controller = (*Controller)(nil)
 
 func (c *Controller) Status(ctx context.Context, sandboxID string) (*api.ControllerStatusResponse, error) {
-	//TODO implement me
-	panic("implement me")
+	sandbox, err := c.sandboxStore.Get(sandboxID)
+	if err != nil {
+		return nil, fmt.Errorf("an error occurred while trying to find sandbox %q: %w",
+			sandboxID, err)
+	}
+	status := sandbox.Status.Get()
```

Comment:
Follow up to my previous comment (https://github.com/containerd/containerd/pull/7470#issuecomment-1317917985)

Should we query task's status here:

```go
container := sandbox.Container
	task, err := container.Task(ctx, nil)
	if err != nil && !errdefs.IsNotFound(err) {
		return nil, fmt.Errorf("failed to get sandbox container task: %w", err)
	}

	var processStatus containerd.ProcessStatus
	if task != nil {
		if taskStatus, err := task.Status(ctx); err != nil {
			if !errdefs.IsNotFound(err) {
				return nil, fmt.Errorf("failed to get task status: %w", err)
			}
			processStatus = containerd.Unknown
		} else {
			processStatus = taskStatus.Status
		}
	}

```